Lib/test/test_genexps.py

```diff
@@ -130,7 +130,7 @@
     Traceback (most recent call last):
       File "<pyshell#4>", line 1, in -toplevel-
         (i for i in 6)
-    TypeError: 'int' object is not iterable
+    TypeError: 'int' object is not iterable. Did you mean: 'range('int')'?
```

Comment:
Single quotes within single quotes does not work.  Just `range(...)` or `range(<int>)` would be better.  Change would apply to other patches below.